src/App.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
+import Header from './components/header/Header.js';
+import Content from './components/content/Content.js';
+import Footer from './components/footer/Footer.js';
+
 import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 function App() {
   return (
-    <div className="App">
-      <h1>Team BLUE</h1>
+    <div>
+      <Header />
+      <Content />
```

Comment:
Content można zrobić tak, żeby wyświetlał `props.children`

```suggestion
      <Content>
          treść
      </Content>
```